sys/mips/mips/elf_machdep.c

```diff
@@ -144,11 +128,18 @@ mips_elf_header_supported(struct image_params * imgp)
 	if (use_cheriabi)
 		return FALSE;
 
-	if (machine == EF_MIPS_MACH_CHERI128)
-		return cheriabi_check_cpu_compatible(128, imgp->execpath);
-	else if (machine == EF_MIPS_MACH_CHERI256)
-		return cheriabi_check_cpu_compatible(256, imgp->execpath);
-	return FALSE;
+	if (machine == EF_MIPS_MACH_CHERI256) {
```

Comment:
It is debatable if we even want to keep this check (or if we do, for how long we want to keep it).

Reply:
It shouldn't cause any maintenance issues, so I would keep it to prevent accidentally running old jenkins binaries.